corehq/apps/domain_migration_flags/api.py

```diff
@@ -53,8 +53,10 @@ def get_migration_status(domain, slug, strict=False):
         return MigrationStatus.NOT_STARTED
 
 
-def migration_in_progress(domain, slug):
-    return get_migration_status(domain, slug) == MigrationStatus.IN_PROGRESS
+def migration_in_progress(domain, slug, any_run=False):
+    """ any_run means a dry run should also be checked """
+    status = get_migration_status(domain, slug)
+    return status == MigrationStatus.IN_PROGRESS or (any_run and status == MigrationStatus.DRY_RUN)
 
 
 @quickcache(['domain'], skip_arg='strict', timeout=60 * 60, memoize_timeout=60)
```

Comment:
Can you update the docstring here to make it clear that it excludes 'dry run' migrations.